### DISCUSSION TITLE
代码中可抽离的逻辑有很多，以判断str是不是string为例

### DISCUSSION THREAD

**Discussion by Sunny-117** [ant-design/ant-design#39664] [Ideas]
<img width="711" alt="image" src="https://user-images.githubusercontent.com/73089592/208444854-c55ac058-4bc3-4580-ac47-eb9c35cbfefd.png">

有计划抽离出公共方法吗？或者现在已有的公共方法存放在哪个目录下？

**Comment by zombieJ** [comment]
抽成方法再引入，func + 路径 uglify 后不一定会小。一般只会封装特别长的。可以参考 rc-util

**Reply by BoyYangzai** [reply]
学到了

**Comment by BoyYangzai** [comment]
小强你没地方蹭pr了是吧
